google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/FlexibleFacetEditor.java

```diff
@@ -78,6 +80,21 @@
   private JPanel dockerfilePanel;
   private AppEngineFlexibleFacetConfiguration facetConfiguration;
 
+  public enum ConfigFile {
```

Comment:
plus, the string value of the enum ("app.yaml", "Dockerfile") doesnt look like its being used